docs/contributor_guide/release.rst

```diff
@@ -30,6 +29,10 @@ done on a clone of `fairlearn/fairlearn` and not on a fork).
 
         :code:`git push -u origin release/v<x.y>.X`
 
+       You may need to temporarily add an exception to the 
```

Comment:
Have we filed an issue to puzzle out what I did wrong in the branch protections?

Reply:
I'm not sure it's really "wrong" - heavily protecting these branches seems like a good idea to me.